src/features/wallet/root/WalletIntroCarousel.tsx

```diff
@@ -49,18 +49,26 @@ const WalletIntroCarousel = () => {
     index: number
     item: CarouselItem
   }) => (
-    <Box borderRadius="m" marginStart="l" height={90} flexDirection="row">
```

Comment:
I think I might have added that height for the dots below it or something? Just want to make sure you tested this change?

Reply:
Here's how it looks on an iPhone 12 Pro Max with no font scaling
![Simulator Screen Shot - iPhone 12 Pro Max - 2021-02-26 at 14 49 42](https://user-images.githubusercontent.com/1760821/109353390-ec717d00-7841-11eb-87a7-b92391c11623.png)

Here's how it looks on an iPhone SE with maximum font scaling.
![Simulator Screen Shot - iPhone SE (2nd generation) - 2021-02-26 at 14 49 43](https://user-images.githubusercontent.com/1760821/109353397-eed3d700-7841-11eb-9d3b-a02b6f86e349.png)

Reply:
Ok cool looks good! Thanks